blazor-toc.html

```diff
@@ -1511,6 +1511,9 @@
 					<li>
 						<a href="/blazor/datagrid/how-to/enable-or-disable-grid">Enable or Disable the Grid Component</a>
 					</li>
+					<li>
+						<a href="/blazor/datagrid/how-to/using-different-edit-mode-for-add-edit-action">Use different EditMode for add and edit action</a>
```

Comment:
move to edting topics https://blazor.syncfusion.com/documentation/datagrid/editing